app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,11 @@
+class PostsController < ApplicationController
+  def index
+    user_id = params[:user_id]
+    puts user_id
```

Comment:
- [ ] OPTIONAL: Please try to always keep your code clean when requesting a Pull. In this case you don't need `user_id = params[:user_id]` and forgot to delete `puts user_id` 😉